.github/workflows/test.yml

```
@@ -60,10 +65,21 @@ jobs:
+
       - name: Run tests
         env:
           PROJECT_ID: ${{ secrets.PROJECT_ID }}
           REGION: ${{ vars.REGION }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
         run: poetry run pytest
+
+      - name: Check Ollama logs (if failure)
+        if: failure()
+        run: docker logs $(docker ps -q --filter "ancestor=ollama/ollama:latest")
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:13: Quote this to prevent word splitting

(shellcheck)